File: c/1180.c
#include "stdio.h"

int main ()
{

  int n, i, x, z;
  int menor, posicao;
  scanf("%d", &n);
  int vetor[n];

  for (i = 0; i < n; i++)
  {

    scanf("%d", &z);
    vetor[i] = z;

  }
  menor = vetor[0];
  for (i = 1; i < n; i++)
  {

    if (vetor[i] < menor)
    {

      menor = vetor[i];
      posicao = i;

    }
  }

  printf("Menor valor: %d\n", menor);
  printf("Posicao: %d\n", posicao);

}

File: go/1180.go
package main

import "fmt"

func main() {
	var n, x, p, v int
	fmt.Scanf("%d", &n)
	fmt.Scanf("%d", &x)
	v = x
	for i := 1; i < n; i++ {
		fmt.Scanf("%d", &x)
		if x < v {
			v = x
			p = i
		}
	}

	fmt.Printf("Menor valor: %d\nPosicao: %d\n", v, p)
}
